src/ui/zcl_abapgit_gui_page_main.clas.abap

```diff
@@ -351,4 +352,40 @@ CLASS ZCL_ABAPGIT_GUI_PAGE_MAIN IMPLEMENTATION.
     ENDCASE.
 
   ENDMETHOD.
+
+
+  METHOD zif_abapgit_gui_page_hotkey~get_hotkey_actions.
+
+    DATA: ls_hotkey_action TYPE zif_abapgit_gui_page_hotkey=>ty_hotkey_action.
+
+    ls_hotkey_action-name   = |Main: Settings|.
+    ls_hotkey_action-action = zif_abapgit_definitions=>gc_action-go_settings.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
+
+    ls_hotkey_action-name   = |Main: Repo overview|.
+    ls_hotkey_action-action = zif_abapgit_definitions=>gc_action-go_repo_overview.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
+
+    ls_hotkey_action-name   = |Main: Refresh|.
+    ls_hotkey_action-action = zif_abapgit_definitions=>gc_action-repo_refresh.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
+
+    ls_hotkey_action-name   = |Main: Pull|.
+    ls_hotkey_action-action = zif_abapgit_definitions=>gc_action-git_pull.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
+
+    ls_hotkey_action-name   = |Main: + Online|.
+    ls_hotkey_action-action = zif_abapgit_definitions=>gc_action-repo_newonline.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
+
+    ls_hotkey_action-name   = |Main: Uninstall|.
+    ls_hotkey_action-action = zif_abapgit_definitions=>gc_action-repo_purge.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
+
+    ls_hotkey_action-name   = |Main: Show diff|.
+    ls_hotkey_action-action = zif_abapgit_definitions=>gc_action-go_diff.
+    INSERT ls_hotkey_action INTO TABLE rt_hotkey_actions.
```

Comment:
no default key per action? I use abapGit across multiple systems, would like not to set it up on each system

Reply:
added default hotkeys